src/model.py

```
@@ -0,0 +1,113 @@
+    def __init__(self, vocab_size, embed_dim=16, num_heads=2, max_seq_len=32, dropout=0.1):
+        super().__init__()
+        self.transformer = nn.ModuleDict(dict(
+            wte = nn.Embedding(vocab_size, embed_dim),
+            wpe = nn.Embedding(max_seq_len, embed_dim),
+            h = nn.ModuleList([Block(embed_dim, num_heads, max_seq_len, dropout)]),
+            ln_f = nn.LayerNorm(embed_dim)
+        ))
+        self.lm_head = nn.Linear(embed_dim, vocab_size, bias=False)
+    
+    def forward(self, idx):
+        B, T = idx.size()
```

Ruff (0.15.15)

[warning] Unpacked variable `B` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)